modules/microvm/virtualization/microvm/netvm.nix

```diff
@@ -62,13 +62,19 @@
 
         microvm = {
           optimize.enable = true;
-          hypervisor = "qemu";
+          hypervisor = {
+            # Use the same machine type as the host
+            x86_64-linux = "cloud-hypervisor";
+            aarch64-linux = "qemu";
+          }
+          .${configHost.nixpkgs.hostPlatform.system};
           shares =
             [
               {
                 tag = "ro-store";
                 source = "/nix/store";
                 mountPoint = "/nix/.ro-store";
+                proto = "virtiofs";
```

Comment:
can you check the extra time delay in booting and size that is incurred by using virtiofs?

Reply:
In my measurements the delay is negligible, less than 1s difference from boot to VM up.

The storage usage grows by 36 kB on AArch64.